daemon/package.json

```diff
@@ -21,9 +21,11 @@
     "@koa/router": "^10.0.0",
     "archiver": "^5.3.1",
     "axios": "^1.1.3",
+    "bufferutil": "^4.0.8",
```

Comment:
Why install it? I didn't find you using it.

Reply:
Same as above.